plugins/formal-verification/testData/diagnostics/good_contracts/is_type_contract.kt

```diff
@@ -22,11 +22,21 @@ open class Foo()
 
 class Bar() : Foo()
 
+@OptIn(ExperimentalContracts::class)
+fun <!VIPER_TEXT!>constructorReturnType<!>(): Boolean {
+    contract {
+        // Kotlin knows that this check will always succeed and marks it as useless, however, we still want to test that
```

Comment:
I think `Suppress` would be easier here to avoid having to maintain it in the diagnostics file.